package.json

```diff
@@ -123,7 +123,7 @@
     "sass-loader": "6.0.6",
     "shortid": "2.2.8",
     "style-loader": "0.20.2",
-    "superdesk-ui-framework": "4.0.3",
+    "superdesk-ui-framework": "^4.0.4",
```

Comment:
imo would be better to avoid `^` there, we had changes in hotfix version which broke builds, the idea to remove that was that we get the updated framework version tested before it's used by clients
cc @tomaskikutis

Reply:
You might have replied about it before, but I forgot - why package-lock isn't enough for hotfixes?

Reply:
it will start breaking builds in superdesk repo but not in superdesk-client-core when there is a change that requires changes in core

Reply:
why would it break builds in main superdesk repo? It also has package-lock

Reply:
or you say that when client-core is being updated from root repo it would update the version then?

Reply:
dependabot will update ui framework and the build will fail, but it won't be visible on client-core repo but rather on some superdesk-cp where we would be using an older version, but which none of the ui team is following

Reply:
I've added a grep-lint rule for this https://github.com/superdesk/superdesk-client-core/pull/4710